build_env.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
By default, tox caches it's environment in the `.tox` directory. If you pass `-r` to tox, it forces recreation of the env.